login-workflow/docs-new/README.md

```diff
@@ -0,0 +1,98 @@
+# React Auth Workflow
+
```

Comment:
I was hoping for a more general overview of the package to set up the stage. Specifically, what can it do ("individual screens and workflows that you can rearrange, configure, and plug into your router management of choice", "default set of translations, but you can overwrite them or add your own")? What it cannot do ("auto handle communication to a backend like BLC, you need to wire up the logic")? What is a component, and what is a screen? Better to also have a link to a running demo somewhere.

Reply:
I know that many of these topics will be more covered in detail in drill-in pages, but I think the root readme should set up some expectations for tech leads who are reading this material and report to their management how will the auth package save their time.